ansible_3par_docker_plugin/tasks/install_prerequisites_on_all.yml

```diff
@@ -4,6 +4,76 @@
       msg: "The plugin properties file does not have the [DEFAULT] back end"
     when: INVENTORY['DEFAULT'] is not defined
 
+  - name: Initialize the variable for mixed configuration
+    set_fact:
+      set_mixed: ''  
+ 
+  - name: Initialize an empty list for drivers
+    set_fact:
+      my_driver_list: []
+  
+  - name: Save list of drivers 
+    set_fact:
+      my_driver_list:  "{{my_driver_list +[INVENTORY[item]['hpedockerplugin_driver']]}}"
+    with_items:
+      - "{{ INVENTORY.keys()}}"
+    register: my_driver_list  
+    
+  - name:  Initialize default driver list
+    set_fact: 
+      default_drivers:
+      - hpedockerplugin.hpe.hpe_3par_iscsi.HPE3PARISCSIDriver
+      - hpedockerplugin.hpe.hpe_3par_fc.HPE3PARFCDriver
+      - hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver
+  
+  - name: Fail when drivers are incorrect or mispelled 
+    fail:
+      msg: "Driver {{item}} is incorrect"
+    with_items:
+      - "{{my_driver_list}}"
+    when: >-
+        "{{item}}" not in default_drivers       
+     
+  - name: Set mixed configuration depending on the driver
+    set_fact:
+      set_mixed: 'true' 
+    when: >-
+      ('hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver'  in  my_driver_list and 'hpedockerplugin.hpe.hpe_3par_iscsi.HPE3PARISCSIDriver' in my_driver_list) or 
+      ('hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver'  in  my_driver_list and 'hpedockerplugin.hpe.hpe_3par_fc.HPE3PARFCDriver' in  my_driver_list )    
+     
+  - name: Fail when default is block and default file is not present
```

Comment:
Re-write is as "DEFAULT section driver is block and DEFAULT_FILE section is not present"